### PR TITLE
SINGA-186 Create Python Tensor class

### PR DESCRIPTION
**Pull request by chonho**

### CODE REVIEW

File: src/python/generate_singa_wrapper.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+!/usr/bin/env bash
```
Comment (nudles) [inline src/python/generate_singa_wrapper.sh]: bash is not available on Windows, need to move these commands to cmake in the future.

File: src/python/generate_singa_wrapper.sh [hunk 2]
```diff
@@ -26,6 +26,8 @@ USR_LOCAL=/home/chonho/local
 
 swig -c++ -python -I../../include singa.i
 
+echo ${SRC_CC[@]}
+
 g++ -fPIC ${SRC_CC[@]} singa_wrap.cxx -shared -o _singa.so \
     -L${USR_LOCAL}/lib -lprotobuf -Wl,-rpath=${USR_LOCAL}/lib \
     -L../../lib -lsinga_core -lsinga_model -lsinga_utils -Wl,-rpath=../../lib \
```
Comment (nudles) [inline src/python/generate_singa_wrapper.sh]: pls replace the hard-code paths, e.g., SINGA_ROOT and USR_LOCAL.
for -I<LIB_PATH>, the LIB_PATH should use the paths found by cmake.

File: src/python/generate_singa_wrapper.sh (hunk 1 above)
Comment (xiezl) [inline src/python/generate_singa_wrapper.sh]: I have already moved the functions in the cmake scripts.
The bash file is only for standalone test.

**Comment by nudles** [comment]
Merged.
TODO:
1. replace hard-code path
2. remove example_xx.py files in python

**Pull request closed without merging**